src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+import Img from './assets/rotateArrow.png';
+import enter from './assets/enter.png';
+
+const tasksContainer = document.getElementById('tasks');
+const rotateArrow = document.getElementById('rotate-arrow');
+const enterIcon = document.getElementById('enter-icon');
+// const titleDiv = document.querySelector('.title');
```

Comment:
* Kindly remove all commented code from your codebase as it makes your code look bulky and unprofessional 🙏